scripts/plotting/tob_wgs_snp_chip_plot_pca/plot_pca_tob_wgs_snp_chip_datasets.py

```diff
@@ -0,0 +1,86 @@
+"""Create PCA plots for the combined
+TOB-WGS/SNP-chip data"""
+
+import subprocess
+from bokeh.io.export import get_screenshot_as_png
+from bokeh.plotting import output_file, save
+import pandas as pd
+import hail as hl
+import click
+from analysis_runner import output_path
+
+SCORES = 'gs://cpg-tob-wgs-main/tob_wgs_snp_chip_variant_pca/v6/scores.ht/'
+EIGENVALUES = 'gs://cpg-tob-wgs-main/tob_wgs_snp_chip_variant_pca/v6/eigenvalues.ht'
+
+
+@click.command()
+def query():  # pylint: disable=too-many-locals
+    """Query script entry point."""
+
+    hl.init(default_reference='GRCh38')
+
+    scores = hl.read_table(SCORES)
+    scores = scores.annotate(snp_chip=(scores.s.contains('snp_chip')))
+    # change from boolean values to what type of data it is
+    expr = (
+        hl.case()
+        .when(
+            (scores.snp_chip),
+            'snp_chip',
+        )
+        .when(
+            (
+                scores.snp_chip  # noqa: E501; pylint: disable=singleton-comparison;
+                == False  # noqa: E712
+            ),
+            'tob_wgs',
+        )
+        .default('NA')
+    )
+    scores = scores.annotate(cohort_sample_codes=expr)
```

Comment:
Do you need the boolean annotation first?

```suggestion
    scores = scores.annotate(cohort_sample_codes=hl.if_else(scores.s.contains('snp_chip'), 'snp_chip', 'tob_wgs'))
```

Reply:
This is a much better way than what I was doing before!